xAnalyzer/xanalyzer.cpp

```diff
@@ -328,14 +328,20 @@ void AnalyzeBytesRange(duint dwEntry, duint dwExit)
 			CallDestination = bii.addr;
 			DbgDisasmFastAt(CallDestination, &cbii);
 			GuiGetDisassembly(CurrentAddress, szDisasmText);
-			GuiGetDisassembly(bii.addr, szJmpDisasmText); // Detect function name on call scheme: CALL -> JMP -> JMP -> API
+			GuiGetDisassembly(CallDestination, szJmpDisasmText); // Detect function name on call scheme: CALL -> JMP -> JMP -> API
 
 			// save data for the argument
 			ai.manual = true;
 			ai.rvaEnd = CurrentAddress - Module::BaseFromAddr(CurrentAddress); // call address is the last		
 			if (Strip_x64dbg_calls(szDisasmText) || (cbii.branch && Strip_x64dbg_calls(szJmpDisasmText)))
 			{
 				szOriginalCharsetAPIFunction = szAPIFunction;
+
+				// Remove Stub suffix from function names if found
+				auto stub = szAPIFunction.find("Stub");
```

Comment:
Do you have an example of this case? I don't recall seeing these suffixes in x64dbg functions names